librust/proc_macro/rust/bridge/ident.rs

```diff
@@ -0,0 +1,68 @@
+use bridge::span::Span;
+use std::convert::TryInto;
+use std::ffi::c_uchar;
+use std::fmt;
+
+extern "C" {
+    fn Ident__new(string: *const c_uchar, len: u64) -> Ident;
```

Comment:
Are you not using `c_size_t` here because it's unstable?

Reply:
Yes, unstable type, hence not in 1.49. Furthermore I thought using `u64` would lead to less errors/unexpected behaviors on some funky architectures.